source/features/list-prs-for-file.tsx

```diff
@@ -61,7 +61,7 @@ function getSingleButton(prNumber: number, _?: number, prs?: number[]): HTMLElem
 const getPrsByFile = cache.function(async (): Promise<Record<string, number[]>> => {
 	const {repository} = await api.v4(`
 		repository() {
-			pullRequests(
+			pullRequests (
```

Comment:
^

Reply:
I added on purpose it looked wrong without a space and the arguments on a new line.

I can revert if you feel it wrong

Reply:
We don't do this anywhere in the code and also doesn't match the function call style in XO. Reverted